### DISCUSSION TITLE
No video is saved even though the VideoWriter node is included

### DISCUSSION THREAD

**Discussion by ejwm** [bonsai-rx/bonsai#1963] [Q&A]
Hi,
I want to record from two cameras simultaneously at the same time as playing a looming stimulus.
Here is my workflow:

![Screenshot 2024-08-07 194626](https://github.com/user-attachments/assets/833ccd4e-8528-4378-9648-ec4c2621fb44)

When I click start the looming stimulus plays and my cameras switch on and look like they're recording but no video files have been saved.
Any help would be appreciated
Thanks,
Eira
[2vid_loom_repeat.zip](https://github.com/user-attachments/files/16532876/2vid_loom_repeat.zip)

**Comment by bruno-f-cruz** [comment]
You need to actually subscribe/start the Observables you create. Add another `Merge` before `RepeatCount` that should do the trick.

**Reply by ejwm** [reply]
I'm still not getting videos after adding another merge

**Reply by bruno-f-cruz** [reply]
Can you try only running this part? 
![image](https://github.com/user-attachments/assets/d896905d-074e-4877-bc86-72cc19d1da32)

With the additional merge?

**Reply by ejwm** [reply]
I don't get any videos even when just running that bit

**Reply by bruno-f-cruz** [reply]
Uhm not sure if there is something wrong with your cameras, but this workflow works fine on my machine:
![image](https://github.com/user-attachments/assets/5e38ae82-e0ce-4b8c-b01c-f9ecae988345)
